### PR TITLE
Bill Situ <Bill.Situ@Oracle.com>

### PR DESCRIPTION
**Pull request by bsitu**
On branch cna/Oracle/CVE-2020-2732
 Changes to be committed:
	modified:   2020/2xxx/CVE-2020-2732.json

### CODE REVIEW

File: 2020/2xxx/CVE-2020-2732.json [hunk 1]
```diff
@@ -1,17 +1,95 @@
 {
```
Comment (cve-team) [inline 2020/2xxx/CVE-2020-2732.json:1]: Invalid JSON file

File: 2020/2xxx/CVE-2020-2732.json [hunk 2]
```diff
@@ -1,18 +1,94 @@
...
```
Comment (cve-team) [inline 2020/2xxx/CVE-2020-2732.json:1]: Invalid JSON file

File: 2020/2xxx/CVE-2020-2732.json [hunk 3]
```diff
@@ -1,18 +1,95 @@
-{
```
Comment (cve-team) [inline 2020/2xxx/CVE-2020-2732.json:1]: Invalid JSON file

Comment (cve-team) [reply 2020/2xxx/CVE-2020-2732.json:1]: Looks like there are excess comma characters for the URLs and vendor name.

Comment (bsitu) [reply 2020/2xxx/CVE-2020-2732.json:1]: Removed extra comma for the URLs in reference, and vendor name.

Comment (cve-team) [inline 2020/2xxx/CVE-2020-2732.json:1]: Duplicate Refs Detected(same URL)

**Comment by bsitu** [comment]
update the duplicate URL (second from the last). Thanks.

**Pull request merged**